packages/components/src/components/button-pure/button-pure.tsx

```diff
@@ -115,7 +123,9 @@ export class ButtonPure {
     improveButtonHandlingForCustomElement(
       this.host,
       () => this.type,
-      () => this.isDisabledOrLoading
+      () => this.isDisabledOrLoading,
+      () => this.name,
```

Comment:
Should these 2 now be rendered on the inner `button` for transparency and maybe a11y, too?